Note: this file is provided as a reference. It is not used when building nginx.

On ARM targets, certain config variables must be changed to accommodate the
architecture. This typically means translating 64 bit values to 32 bit. When
updating nginx, the difference between objs/ngx_auto_config.h and the vendored
srcpkgs/nginx/files/ngx_auto_config.h.armv6l should match this patch. Additional
definitions may need to be added to the vendored config file. If these
definitions contain architecture specific values like those below, they need to
be translated to their 32 bit equivalents.

--- objs/ngx_auto_config.h.orig	2019-02-09 22:49:23.758277257 -0800
+++ objs/ngx_auto_config.h	2019-02-09 23:13:01.326692988 -0800
@@ -80,13 +80,13 @@
 #endif
 
 
-#ifndef NGX_HAVE_NONALIGNED
-#define NGX_HAVE_NONALIGNED  1
+#ifndef NGX_ALIGNMENT
+#define NGX_ALIGNMENT  16
 #endif
 
 
 #ifndef NGX_CPU_CACHE_LINE
-#define NGX_CPU_CACHE_LINE  64
+#define NGX_CPU_CACHE_LINE  32
 #endif
 
 
@@ -204,7 +204,7 @@
 
 
 #ifndef NGX_PTR_SIZE
-#define NGX_PTR_SIZE  8
+#define NGX_PTR_SIZE  4
 #endif
 
 
@@ -219,12 +219,12 @@
 
 
 #ifndef NGX_MAX_SIZE_T_VALUE
-#define NGX_MAX_SIZE_T_VALUE  9223372036854775807LL
+#define NGX_MAX_SIZE_T_VALUE  2147483647
 #endif
 
 
 #ifndef NGX_SIZE_T_LEN
-#define NGX_SIZE_T_LEN  (sizeof("-9223372036854775808") - 1)
+#define NGX_SIZE_T_LEN  (sizeof("-2147483648") - 1)
 #endif
 
 
@@ -239,17 +239,17 @@
 
 
 #ifndef NGX_TIME_T_SIZE
-#define NGX_TIME_T_SIZE  8
+#define NGX_TIME_T_SIZE  4
 #endif
 
 
 #ifndef NGX_TIME_T_LEN
-#define NGX_TIME_T_LEN  (sizeof("-9223372036854775808") - 1)
+#define NGX_TIME_T_LEN  (sizeof("-2147483648") - 1)
 #endif
 
 
 #ifndef NGX_MAX_TIME_T_VALUE
-#define NGX_MAX_TIME_T_VALUE  9223372036854775807LL
+#define NGX_MAX_TIME_T_VALUE  2147483647
 #endif
 
 
@@ -343,11 +343,6 @@
 #endif
 
 
-#ifndef NGX_HAVE_LEVEL1_DCACHE_LINESIZE
-#define NGX_HAVE_LEVEL1_DCACHE_LINESIZE  1
-#endif
-
-
 #ifndef NGX_HAVE_OPENAT
 #define NGX_HAVE_OPENAT  1
 #endif
